puphpet/config.yaml.template

```diff
@@ -147,15 +174,41 @@ php:
     composer_home: ''
     modules:
         php:
-            - cli
-            - intl
-            - mcrypt
-            - memcached
-            - sqlite
-            - curl
+            - cli # Needed for shells
+            #- curl
+            #- gd
+            #- geoip
+            #- imagick
+            - intl # Required by Cake 3.
+            - mbstring # Required by Cake 3.
+            - mcrypt # Required by Cake 3.
+            #- memcache
+            #- memcached
+            #- mysql
+            #- readline
+            #- sqlite
```

Comment:
Having SQLite this installed by default may make running tests the the VM faster/easier. I'll need to look into it a bit more.